### PR TITLE
Tabular: tabular_prediction.py fit args cleanup + README.md update

### PR DESCRIPTION
**Pull request by Innixma**
*Issue #, if available:*

*Description of changes:*

Tabular: tabular_prediction.py fit args cleanup + README.md update

By submitting this pull request, I confirm that you can use, modify, copy, and redistribute this contribution, under the terms of your choice.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,7 +15,10 @@ AutoGluon automates machine learning tasks enabling you to easily achieve strong
 ## Example
 
 ```python
-# First install package from terminal:  pip install mxnet autogluon
+# First install package from terminal:
+# python3 -m pip install --upgrade pip
+# python3 -m pip install --upgrade "mxnet<2.0.0"
+# python3 -m pip install autogluon
```
Comment (jwmueller) [inline README.md:21]: put these all on the same line?

**Review by jwmueller — APPROVED** [review]
looks good

**Comment by szha** [comment]
Job PR-689-1 is done. 
Docs are uploaded to http://autogluon-staging.s3-website-us-west-2.amazonaws.com/PR-689/1/index.html

File: README.md (hunk 1 above)
Comment (Innixma) [reply README.md:21]: Can't because mxnet must be installed before autogluon, and pip must be updated to get mxnet 1.7, so they need to be separate lines.

**Pull request merged**